src/SDKs/Resource/Management.ResourceManager/Microsoft.Azure.Management.ResourceManager.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>Microsoft.Azure.Management.ResourceManager</PackageId>
     <Description>Provides resource group and resource management capabilities for Microsoft Azure.</Description>
     <AssemblyName>Microsoft.Azure.Management.ResourceManager</AssemblyName>
-    <VersionPrefix>1.5.0-preview</VersionPrefix>
+    <VersionPrefix>2.0.0-preview</VersionPrefix>
```

Comment:
@TianoMS What about assembly version in AssemblyInfo.cs? Also I noticed other RPs tend to use major version here. Since we have a big change, should we also kick off a new major version?